packages/backend-core/src/utils/utils.ts

```diff
@@ -38,7 +30,7 @@ export async function resolveAppUrl(ctx: Ctx) {
   const appUrl = ctx.path.split("/")[2]
   let possibleAppUrl = `/${appUrl.toLowerCase()}`
 
-  let tenantId: string | null = tenancy.getTenantId()
+  let tenantId: string | null = context.getTenantId()
```

Comment:
Something I've been thinking about - should really get rid of all context based stuff from tenancy, could be a worthwhile refactor moving it all over at some point.

Reply:
It was done more out of necessity because of some circular deps in backend-core. It's just externally now that the context package is re-added to tenancy for backwards compatibility:
https://github.com/Budibase/budibase/blob/bd09914e4ef41da73884817d79e27b3891bc2882/packages/backend-core/src/index.ts#L33-L36

It wouldn't be that big of a job now to just remove that, build and fix the fallout.